paymentsheet/src/test/java/com/stripe/android/paymentsheet/viewmodels/PaymentOptionsStateMapperTest.kt

```diff
@@ -74,22 +64,22 @@ class PaymentOptionsStateMapperTest {
             isNotPaymentFlow = false,
         )
 
-        val state = mapper.createStateAndObserve()
-
-        val cards = PaymentMethodFixtures.createCards(2)
-        paymentMethodsFlow.value = cards
-        initialSelectionFlow.value = SavedSelection.PaymentMethod(id = cards.first().id!!)
-        googlePayStateFlow.value = GooglePayState.Available
-        isLinkEnabledFlow.value = true
-
-        assertThat(state.value?.items).containsNoneOf(
-            PaymentOptionsItem.GooglePay,
-            PaymentOptionsItem.Link,
-        )
+        mapper().test {
+            val cards = PaymentMethodFixtures.createCards(2)
+            paymentMethodsFlow.value = cards
+            initialSelectionFlow.value = SavedSelection.PaymentMethod(id = cards.first().id!!)
+            googlePayStateFlow.value = GooglePayState.Available
+            isLinkEnabledFlow.value = true
+
+            assertThat(expectMostRecentItem()?.items).containsNoneOf(
```

Comment:
I don't love the usage of `expectMostRecentItem`

Reply:
I think we will be removing PaymentOptionsStateMapper in the near future.